File: src/main/java/com/leijendary/spring/boot/template/api/v1/data/SampleSearchResponse.java
package com.leijendary.spring.boot.template.api.v1.data;

import java.io.Serializable;
import java.time.OffsetDateTime;
import java.util.UUID;

import lombok.Data;

@Data
public class SampleSearchResponse implements Serializable {

    private UUID id;
    private String column1;
    private String column2;
    private String name;
    private String description;
    private OffsetDateTime createdAt;
}

File: src/main/kotlin/com/leijendary/spring/boot/template/api/v1/data/SampleSearchResponse.kt
package com.leijendary.spring.boot.template.api.v1.data

import java.io.Serializable
import java.time.OffsetDateTime
import java.util.*

data class SampleSearchResponse(
    val id: UUID,
    val column1: String,
    val column2: String,
    val name: String,
    val description: String,
    val createdAt: OffsetDateTime,
) : Serializable